### ISSUE TITLE
번역시 합의가 필요한 용어들

### ISSUE THREAD

**Issue by alanleedev** [alanleedev/KoreaSecurityApps#8]
@kunggom 님의 [제안](https://github.com/alanleedev/KoreaSecurityApps/issues/7#issuecomment-1386965373)에 따라 extension은 크롬에서 사용하는 확장 프로그램으로 상요하도록 하겠습니다.
다른 합의가 필요한 용어가 있다면 아래 의견 주세요.

**Comment by kunggom** [comment]
아, 한 가지 더 있네요.
`license`라는 단어는 [외래어 표기법에 따르면 `라이선스`라고 표기해야](https://www.korean.go.kr/news/index.jsp?control=page&part=view&idx=10891) 한다고 합니다.
제가 수정한 부분에서는 모두 바꾸었지만, 원래 올라왔던 번역문에서는 `라이센스`라고 표기된 곳이 많아서 추후 번역에 참고하고자 여기에 적어 둡니다.

**Comment by kunggom** [comment]
제가 `도우미 (helper) 애플리케이션 악용하기` 부분을 다듬다가 생각난 것이 있어서 제안드립니다.

해당 부분에는 `integrity level`이라는 용어가 등장하는데, [Microsoft 측에서 제공하는 한국어 문서](https://learn.microsoft.com/ko-kr/windows/win32/secauthz/mandatory-integrity-control)에서는 이걸 `무결성 수준`이라고 직역하고 있습니다. 그런데 제가 본 [다른 자료](https://www.slideshare.net/hajimaru1/01windows-20150613)에서는 이걸 `신뢰도`라고 번역하고 있습니다. 개인적으로는 `신뢰도`라는 번역이 조금 더 마음에 듭니다만, 구글 검색 결과 이 번역어를 사용하는 경우는 비교적 드문 것 같습니다.

비슷한 경우로, 문서 상에서 위 용어와 인접하여 등장하는 `security descriptor`도 있습니다. 한국정보통신기술협회의 정보통신용어사전에서는 이 용어를 [`보안 기술자`](http://terms.tta.or.kr/mobile/dictionaryView.do?subject=security%20descriptor)라고 번역했지만, Microsoft의 용어집에서는 [`보안 설명자`](https://learn.microsoft.com/ko-kr/windows/win32/secgloss/s-gly#_security_security_descriptor_gly)라고 번역하고 있었습니다.

그래서 조금 고민해 보았는데요, 아무래도 이런 금융 보안 프로그램은 기본적으로 윈도우 운영체제에서 사용할 것을 전제로 한 경우가 많습니다. 그래서 이렇게 **번역이 갈릴 수 있는 용어의 경우에는 MS에서 제공하는 한국어 문서에 등장하는 번역어를 우선으로 하되, 영어 표기를 괄호 안에 넣어서 표시하고 필요에 따라 간혹 문단 아래에 역주를 별도로 추가**한다는 원칙을 사용하는 것이 어떨까 합니다.

예를 들면 다음과 같은 방식입니다.

> 무결성 수준(integrity level)
>
>   (역주: ‘integrity level’이라는 용어는 [Microsoft의 한국어 문서](https://learn.microsoft.com/ko-kr/windows/win32/secauthz/mandatory-integrity-control)에서는 “무결성 수준”으로 직역되어 있지만, [일부 자료](https://www.slideshare.net/hajimaru1/01windows-20150613)에서는 “신뢰도”라고 번역하는 경우도 있습니다.)

**Comment by alanleedev** [comment]
네 좋습니다. 원문을 같이 표기하는게 아무래도 해석상 오해의 여지를 줄일 수 있겠죠. 저도 마이크로소프트의 한글 번역이 썩 맘에 들지는 않지만 제품의 실질적 주인이라 그쪽에서 사용하는 공식용어를 사용하는 것이 더 맞을 것 같긴하네요.

**Comment by KENNYSOFT** [comment]
'한글'과 '한국어'를 구분했으면 합니다. 현재 2편 내용 중 거의 대부분의 경우는 '한글' 보다는 '한국어' 가 맞습니다.

다만 '한글 특허 문서'는 '한국 특허 문서'로 하는 것이 좋겠습니다.

**Comment by alanleedev** [comment]
@KENNYSOFT 이건 일단 제가 지금 일괄 수정할께요.

**Comment by kunggom** [comment]
맨 첫번째 글의 마지막 부분 다듬기 작업을 하다가 생각난 점이 있어서 제안드립니다.

보안 취약점을 `Report`하는 것을 현재는 `보고`라고 번역하고 있는데, 이를 `신고`로 번역하는 것이 좋겠다고 생각합니다.

- Google 검색 결과 `취약점 신고`의 검색결과가 가장 많으므로, 다른 표현에 비해 훨씬 널리 쓰이는 표현으로 판단됨
  - `취약점 신고`: 약 170만 건
  - `취약점 보고`: 약 93만 8천 건
  - `취약점 제보`: 약 24만 2천 건
- 첫번째 글에 나왔던 KrCERT의 [웹페이지](https://www.krcert.or.kr/consult/software/vulnerability.do)를 보면 `보안 취약점 신고`라고 하고 있음

**Comment by kunggom** [comment]
외래어 표기법에 대한 좋은 문서를 찾았습니다.
가급적 이 문서의 내용을 따르는 건 어떨까 제안합니다.

- [헷갈리는 외래어, 어떻게 써야 할까요?](https://meetup.nhncloud.com/posts/355)

**Comment by alanleedev** [comment]
네. 이미 누군가 좋은 기준 제시한 것 있으면 활용하면 좋을 것 같네요.
